AdyenCard/Form/FormCardSecurityCodeItem.swift

```diff
@@ -69,6 +74,20 @@ extension Array where Element == CardBrand {
         }
     }
     
+    /// At the moment, due to UI concerns, we won't hide cvc/exp date fields even when response is hidden
+    /// We will make them optional.
+    var isExpiryDateOptional: Bool {
```

Comment:
what if we do this
```swift
    /// Indicates the requirement level of a field.
    public enum RequirementPolicy: String, Decodable {

        /// Field is required.
        case required

        /// Field is optional.
        case optional

        /// Field should not be asked.
        case hidden
        
        var isOptional: Bool {
            self != .required
        }
    }

extension Array where Element == CardBrand {
    var isCVCOptional: Bool {
        isEmpty ? false : lazy.allSatisfy(\.cvcPolicy.isOptional)
    }

    var isExpiryDateOptional: Bool {
        isEmpty ? false : lazy.allSatisfy(\.cvcPolicy.isOptional)
    }
}

Reply:
DO you mean move the extension to CardBrand? If so, makes sense :)